mobile/cmp/input/Select.js

```diff
@@ -42,18 +47,25 @@ Select.propTypes = {
     createMessageFn: PT.func,
 
     /**
-     *  True to accept and commit input values not present in options or returned by a query.
-     *  Should be used with caution, as mobile keyboard can cause undesirable results.
-     * */
+     * True to accept and commit input values not present in options or returned by a query.
+     * Usually used with enableFullscreen, to ensure access to the mobile device keyboard.
+     */
     enableCreate: PT.bool,
 
     /**
      * True to enable type-to-search keyboard input. Defaults to false to disable keyboard input,
-     * showing the dropdown menu on click. Should be used with caution, as mobile keyboard can
-     * cause undesirable results.
+     * showing the dropdown menu on click. Usually used with enableFullscreen, to ensure access
+     * to the mobile device keyboard.
      */
     enableFilter: PT.bool,
 
+    /**
+     * True to render the select control in a full-screen modal dialog when focused.
+     * Usually applicable when enableCreate|enableFilter, as we can guarantee the control
```

Comment:
Suggest replacing "Usually..." in these comments with "Recommended for use with..."